package.json

```diff
@@ -65,11 +65,15 @@
     ]
   },
   "dependencies": {
+    "electron-json-storage": "^4.4.0",
     "github-markdown-css": "4.0.0",
+    "github-markdown-dracula-css": "^4.0.2",
```

Comment:
Look out for the syntax of the package versions. I require always *the exact* version to be (almost) sure that any commit/version can be built in the future - maybe even in years to come.

Reply:
Thank you! I have changed this also

Reply:
I removed an accedently added package, it should all be okay now